src/test/unit_tests/test_namespace_fs.js

```diff
@@ -472,6 +474,22 @@ mocha.describe('namespace_fs', function() {
 
         });
 
+        mocha.it('delete dir object without last slash - a/b/c', async function() {
```

Comment:
minor (missing the first slash in the title)
```suggestion
        mocha.it('delete dir object without last slash - /a/b/c', async function() {
```